plugins/modules/system/xfconf_info.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: xfconf_info
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Retrieve XFCE4 configurations
+description:
+  - This module allows retrieving Xfce 4 configurations via xfconf-query.
```

Comment:
```suggestion
  - This module allows retrieving Xfce 4 configurations with the help of C(xfconf-query).
```
We're trying to avoid Latin words :)

Reply:
Ok, will send another PR fixing that in xfconf - this one came from there.